client/src/index.css

```diff
@@ -1,13 +1,3 @@
 body {
```

Comment:
I see you are overriding a browser default (which is good imo). You may want to read a bit of this article on the topic. There is some common ones like [Reset CSS](https://meyerweb.com/eric/tools/css/reset/index.html) that you can copy into your index.css if you want. Optional, I don't always do it.

Reply:
The margin: 0 was all that was left from the default CRA index.css. I still don't know how I want to deal with styling so I'm just going to leave it as is.